engineering_tools/carma

```diff
@@ -204,7 +204,26 @@ carma-config__install() {
     echo "Cleaning up temporary container..."
     docker rm carma-config-tmp
 }
+carma__record() {
+    last_modified_file="$(find "/opt/carma/logs" -type f \( -name \*.bag.active -o -name \*.bag \) -printf "%T@ %p\0" | awk 'BEGIN {RS="\0";} {if (NR==1){minmtime=$1; $1=""; lastmodified=$0;} else if (minmtime<$1){minmtime=$1; $1=""; lastmodified=$0;}} END{print substr(lastmodified,2)}')"
```

Comment:
I think some commends explaining this complex one liner are needed, since it is quite hard to read